middleware/package.json

```diff
@@ -27,7 +27,8 @@
   "author": "TODO",
   "dependencies": {
     "@projectriff/message": "^1.0.0",
-    "@salesforce/salesforce-sdk": "^1.3.0",
+    "@salesforce/core": "^2.10.0",
```

Comment:
We don't need thing anymore do we?

Reply:
Good catch!  Yeah, we don't need it now!